ui/00_message/ui/system_log.xml

```diff
@@ -393,139 +393,139 @@
   </message>
   <message>
     <key>syslog_craft_consume_item</key>
-    <text xml:space="preserve">&lt;NAME ITEM&gt;を&lt;VAL&gt;個、消費しました</text>
+    <text xml:space="preserve">&lt;NAME ITEM&gt; &lt;VAL&gt; was consumed.</text>
```

Comment:
Might want to change the order of the string, for eg. below (number of items were consumed)

<text xml:space="preserve">&lt;VAL&gt; &lt;NAME ITEM&gt;'s were consumed.</text>